src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -165,7 +161,7 @@ private function getMetadata($type, $entity)
                     return false;
                 }
 
-                return !$param->getClass() || $param->getClass()->getName() !== 'Twig_Environment';
+                return !$param->getClass() || !in_array($param->getClass()->getName(), array('Twig_Environment', 'Twig\Twig'));
```

Comment:
Actually, this logic could be improved to avoid relying on typehints or param names (btw, the check on param name is wrong, as the name is not relevant). I will send a PR.